apollo-federation/src/operation/mod.rs

```diff
@@ -3955,7 +3955,7 @@ impl TryFrom<Operation> for Valid<executable::ExecutableDocument> {
         document.fragments = fragments;
         document.operations.insert(operation);
         coerce_executable_values(value.schema.schema(), &mut document);
-        Ok(document.validate(value.schema.schema())?)
+        Ok(Valid::assume_valid(document))
```

Comment:
We should not do `assume_valid` inside the `TryFrom` implementation. If we do this we should change the impl to `TryFrom<Operation> for executable::ExecutableDocument`.